src/core/a-entity.js

```diff
@@ -423,15 +425,15 @@ var proto = Object.create(ANode.prototype, {
   },
 
   removeComponent: {
-    value: function (name) {
+    value: function (name, force) {
```

Comment:
The problem I was solving was that after a call to `reload`, `canvas` and `look-controls` were still relying on the injected `<canvas>` element that was no longer in the tree. The most maintainable approach I saw was to follow the pattern that the rest of `reload` follows and simulate how the DOM was originally constructed as closely as possible. That meant resetting  `<a-scene>`'s `components` to empty and reinitializing everything.   I tried to make the process as close to the original initialization as possible so that the component level wouldn't need to know the difference between initializing and reinitializing.

So while in general removing `<canvas>` doesn't make sense, it actually did in this one weird case.